dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -1012,6 +1012,24 @@ void EnableLightDismiss()
             }
         }
 
+        [TestMethod]
+        public void VerifyTeachingTipButtonsNameAutomationProperty()
+        {
+            using(var setup = new TestSetupHelper(new[] { "TeachingTip Tests", "TeachingTip Test" }))
+            {
+                elements = new TeachingTipTestPageElements();
+                SetActionButtonContentTo("Small text");
+
+                elements.GetUpdateActionNameButton().Invoke();
```

Comment:
I don't think this is testing what we want to test.

What we need to verify is:
If ActionButtonContent is set, the AutomationName of the Button matches that. You can use UIA to find the Button in the UIA tree and verify that its Automation Name is as expected.